package by.cp.feedback.mechanism.bot.exception

class QuestionNotFoundException : FeedbackBotException("Question not found in reply")